python/eos/analysis.py

```diff
@@ -653,14 +653,55 @@ def sample_nested(self, bound='multi', nlive=250, dlogz=1.0, maxiter=None, seed=
         :type maxiter: int, optional
         :param seed: The seed used to initialize the Mersenne Twister pseudo-random number generator.
         :type seed: {None, int, array_like[ints], SeedSequence}, optional
+        :param save_intermediate: If set to True, the intemediate dynesty sampler results are stored in each loop iteration.
+        :type save_intermediate: bool, optional
+        :param checkpoint_every: The number of seconds between checkpoints at which the intermediate dynesty sampler results are stored.
```

Comment:
I like the checkpointing. Not so sure that I like the naming.

Reply:
What about `backup` and `backup_frequency`?

Reply:
How about just a `checkpoint_interval` (float, larger zero)? If `None`, no intermediate results are saved.